pamelalozano/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -45,6 +48,16 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       return;
     }
 
+    //Storing in database
+    Entity commentEntity = new Entity("Comment");
```

Comment:
You could make a pretty handy converter to create Entitys based on Comments, ie:

`Entity commentEntity = createEntity(comment);`